frontend/www/js/omegaup/components/course/Details.vue

```diff
@@ -174,8 +176,22 @@
       </div>
     </div>
 
+    <div v-if="!loggedIn">
+      <div class="card">
+        <div class="card-header">
+          <h5 class="text-center">
+            <a
+              :href="`/login/?redirect=${encodeURI(window.location.pathname)}`"
```

Comment:
```suggestion
              :href="`/login/?redirect=${encodeURIComponent(window.location.pathname)}`"
```

cuando una cadena se va a usar como _parte_ de un URI (parámetro, parte de la ruta, etc.), se recomienda usar [`encodeURIComponent()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURIComponent#Description) para evitar que se cuelen caracteres como `&` que tienen significado especial en esas partes.

Reply:
Listo